config/application.rb

```diff
@@ -73,7 +73,12 @@ class Application < Rails::Application
     config.samson.docker = ActiveSupport::OrderedOptions.new
     config.samson.docker.registry = ENV['DOCKER_REGISTRY'].presence
 
-    config.samson.uri = URI( ENV["DEFAULT_URL"] || 'http://localhost:3000' )
+    config.samson.uri = URI(ENV["DEFAULT_URL"] || 'http://localhost:9080')
```

Comment:
Why the port change to 9080? Should probably be 3000, since that's still the development default.